Validate that at least one provider exists in ext-auth config (#1294)
kyma-project/api-gateway: docs/user/custom-resources/apirule/v2alpha1/04-10-apirule-custom-resource.md

| **rules.jwt.authorizations.audiences**           |  **NO**  | Specifies the list of audiences required for the JWT.                                                                                                                                                                                                                                 | None                                                                                                                  |
| **rules.extAuth**                                |  **NO**  | Specifies the Istio External Authorization access strategy.                                                                                                                                                                                                                           | Must exists if noAuth and jwt are not specified.                                                                      |

> | **rules.extAuth.authorizers**                    | **YES**  | Specifies the Istio External Authorization authorizers. In case extAuth is configured, at least one needs to be present.                                                                                                                                                                                                                             | Validated that the provider exists in Istio external authorization providers.                                         |

Reviewer: ```suggestion
| **rules.extAuth.authorizers**                    | **YES**  | Specifies the Istio External Authorization authorizers. In case extAuth is configured, at least one must be present.                                                                                                                                                                                                                             | Validated that the provider exists in Istio external authorization providers.                                         |
```
